### DISCUSSION TITLE
Mesh with hex elements conforming the interface

### DISCUSSION THREAD

**Discussion by schakrabortygithub** [neperfepx/neper#734] [Q&A] [answered]
Hi,

Is it possible to generate the mesh of a polycrystalline domain using hex element such that the mesh will conform with the Grain Boundary?
I am able to do this using Tet element but with Hex elemet the mesh is of low quality and does not conform with the grain boundary. This is what I am getting:
<img width="1159" alt="Screen Shot 2023-08-10 at 12 03 37 PM" src="https://github.com/neperfepx/neper/assets/35871741/5b3bc965-a8f0-446a-81ff-60682212cb21">

And this is what I want (Generated using Cubit):

<img width="573" alt="Screen Shot 2023-08-15 at 3 23 22 PM" src="https://github.com/neperfepx/neper/assets/35871741/842cd700-cfa6-4f24-a2bb-513dc3c61497">

Best
Subhendu Chakraborty
Los Alamos National Laboratory

**Comment by rquey** [comment, marked as answer]
See this discussion: https://github.com/neperfepx/neper/discussions/98#discussioncomment-318210.
